G/GibbsSeaWater/build_tarballs.jl

```diff
@@ -1,22 +1,29 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
 name = "GibbsSeaWater"
-version = v"3.5.0"
+version = v"3.5.1"
 
 # Collection of sources required to build GibbsSeaWater
 # note 3.05.0-4 is too old to be cross-compiled on FreeBSD
 sources = [
-    "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip" =>
-    "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c",
+    ArchiveSource(
+        "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip", "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c"),
 ]
 
+
+
 # Bash recipe for building across all platforms
 script = raw"""
+if [[ "${target}" == aarch64-apple-* ]]; then
+    # Linking libomp requires the function `__divdc3`, which is implemented in
+    # `libclang_rt.osx.a` from LLVM compiler-rt.
+    LDFLAGS="-L${libdir}/darwin -lclang_rt.osx"
+fi
 cd $WORKSPACE/srcdir/GSW-C-*
 cc $CFLAGS -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
-cc $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm
+cc $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm 
```

Comment:
This introduced an extra whitespace
```suggestion
cc $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm
```